stdlib/experimental/influxdb/influxdb.go

```diff
@@ -43,27 +43,39 @@ func api(ctx context.Context, a values.Object) (values.Value, error) {
 		validator fluxurl.Validator
 		client    fluxhttp.Client
 		err       error
+		ok        bool
+		args      = interpreter.NewArguments(a)
+		host      string
 	)
 	{
 		if validator, err = deps.URLValidator(); err != nil {
 			return nil, fluxerrors.New(codes.Internal, "missing dependencies")
 		}
-		if client, err = deps.HTTPClient(); err != nil {
+
+		if host, ok, err = args.GetString("host"); err != nil {
+			return nil, err
+		} else if !ok {
+			return nil, fluxerrors.New(codes.Invalid, `keyword argument "host" is required when executing outside InfluxDB`)
```

Comment:
I am pretty sure that we will hit this path if `host` is an empty string, so I think the first branch below is dead code. I think this is okay though, since it's defensive.

(There is a separate implementation of this that is used for cloud)

Reply:
For my own clarity, do you know where the other implementation is? I ran into the cloud vs. oss difference while I was debugging this, and I couldn't find the version where specifying the host is optional. Maybe I can move this fix there.